client/src/components/TimetableSwitcher/TimetableSwitcher.tsx

```diff
@@ -28,7 +31,10 @@ const TimetableSwitcher = memo(() => {
 	const activeTimetableName = useSelector(
 		(state: RootState) => state.timetable.active,
 	);
-
+    const timetable = useSelector(
+        (state: RootState) => selectFilteredTimetable(state),
```

Comment:
This is an anti-pattern. The `selectFilteredTimetable` is a selector in itself
```suggestion
        const timetable = useSelector(selectFilteredTimetable);
```